backend/spec/controller_exports_spec.rb

```diff
@@ -55,16 +41,6 @@
   end
 
 
-  it "lets you export a software in EAC-CPF" do
-    id = create(:json_agent_software_full_subrec).id
-    get "/repositories/#{$repo_id}/archival_contexts/softwares/#{id}.xml"
-    resp = last_response.body
-    expect(resp).to include("<eac-cpf")
-    expect(resp).to include("<control>")
```

Comment:
`expect(resp).to include("<control>")` should be removed once this test is restored... we'll be exporting an empty `<control/>` always now, so there's no point in testing that.